tests/recordRequests.js

```diff
@@ -41,11 +42,20 @@ function getRequestHash(input, init) {
   return hash.digest();
 }
 
+const originalFetch = globalThis.fetch;
+const passthroughUrls = [];
+
 if (process.env.NOCK_ENV === `record`) {
   const insertNockStatement = db.prepare(`INSERT OR REPLACE INTO nocks (hash, body, headers, status) VALUES (?, ?, jsonb(?), ?)`);
-  const realFetch = globalThis.fetch;
+
   globalThis.fetch = async (input, init) => {
-    const response = await realFetch(input, init);
+    if (passthroughUrls.some(passThrough => input.toString().startsWith(passThrough)))
+      return originalFetch(input, init);
+    if (process.env.BLOCK_SIGSTORE_TUF_REQUESTS && input.toString().startsWith(`https://tuf-repo-cdn.sigstore.dev`))
```

Comment:
an allow list pattern would be better than a deny list here.

Reply:
To be honest, I would like to avoid sinking more time into this, especially if a change does not make things worse than before. Or is there something I’m overlooking?

Reply:
This is just a non blocking comment, especially if the pattern used here is pre-existing.